### PR TITLE
some design cleanup

### PR DESCRIPTION
**Pull request by jbibla**
Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by Bitcoinera** [comment]
Missing changelog, unit tests failing... a number of things

File: app/src/components/governance/LiProposal.vue [hunk 1]
```diff
@@ -18,15 +18,15 @@
         </h3>
       </div>
       <span v-if="proposal.creationTime" class="time">{{
-        proposal.creationTime | moment
```
Comment (Bitcoinera) [inline app/src/components/governance/LiProposal.vue:21]: This should be done in another PR. One only dedicated to dates like the one Mario is doing

File: app/src/components/governance/PageProposals.vue [hunk 2]
```diff
@@ -16,7 +16,7 @@
             @click.native="onPropose"
           />
           <TmBtn
-            v-if="currentNetwork.network_type === `cosmos`"
+            v-if="currentNetwork.slug === `cosmos-hub`"
```
Comment (Bitcoinera) [inline app/src/components/governance/PageProposals.vue:19]: ? Why?

File: app/src/components/governance/ParticipantList.vue [hunk 3]
```diff
@@ -78,10 +89,6 @@ export default {
     },
   },
   methods: {
-    getParticipantName(participant) {
-      const name = participant.name
-      return name.length > 25 ? formatAddress(name) : name || `n/a`
-    },
```
Comment (Bitcoinera) [inline app/src/components/governance/ParticipantList.vue:84]: It is going to be too long, no?

Comment (Bitcoinera) [reply app/src/components/governance/ParticipantList.vue:84]: Look, this is how it looks on mobile:

image

Comment (Bitcoinera) [reply app/src/components/governance/ParticipantList.vue:84]: Ok, we'll do it in another PR

Comment (jbibla) [reply app/src/components/governance/ParticipantList.vue:84]: we'll leave this now. in the API if there is no name, it should be null, and the front end will display + format the address when there is no name.

File: app/src/components/governance/Timeline.vue [hunk 4]
```diff
@@ -60,6 +72,13 @@ section {
   background: var(--app-bg);
 }
 
+.done.phase::before {
+  content: "\2714";
```
Comment (Bitcoinera) [inline app/src/components/governance/Timeline.vue:76]: :eyes:

**Review by Bitcoinera — APPROVED** [review]
